dev/ush/make_ee2_links.sh

```diff
@@ -0,0 +1,338 @@
+                    for tile in {1..6}; do
+                        link_file "${system_prefix}.t${cyc}z.cubed_sphere_grid_ratminc.tile${tile}.nc" "${system_prefix}.t${cyc}z.recentered_jedi_increment.atm.i006.tile${tile}.nc"
+                    done
+                fi
+                # abias
+                for abias_type in abias abias_air abias_int abias_nc; do
+                    if [[ -f "${system_prefix}.t${cyc}z.${abias_type}.ensmean" ]]; then
+                        link_file "${system_prefix}.t${cyc}z.${abias_type}.ensmean" "${system_prefix}.t${cyc}z.${abias_type}.ensmean.txt"
+                    fi
+                done
+                # stats
+                for stat_type in cnvstat gsistat oznstat radstat; do
+                    if [[ -f "${system_prefix}.t${cyc}z.${stat_type}.ensmean" ]]; then
+                        link_file "${system_prefix}.t${cyc}z.${stat_type}.ensmean" "${system_prefix}.t${cyc}z.${stat_type}.ensmean.tar"
+                    fi
+                done
+                if [[ -f "${system_prefix}.t${cyc}z.enkfstat" ]]; then
+                    link_file "${system_prefix}.t${cyc}z.enkfstat" "${system_prefix}.t${cyc}z.enkfstat.txt"
+                fi
+                # surface increments
+                for inc_time in 003 006 009; do
+                    if [[ -f "${system_prefix}.t${cyc}z.sfci${inc_time}.nc" ]]; then
+                        link_file "${system_prefix}.t${cyc}z.sfci${inc_time}.nc" "${system_prefix}.t${cyc}z.increment.sfc.i${inc_time}.nc"
+                    fi
+                done
+                # sfc_inc tile links
+                if [[ -f "sfc_inc.tile1.nc" ]]; then
+                    for tile in {1..6}; do
+                        link_file "sfc_inc.tile${tile}.nc" "increment.sfc.i006.tile${tile}.nc"
+                    done
+                fi
+            fi
+            # ocean
+            if [[ -d "${cwd}/${dir}/${cyc}/${mem}/analysis/ocean" ]]; then
+                cd "${cwd}/${dir}/${cyc}/${mem}/analysis/ocean"
+                if [[ -f "${system_prefix}.t${cyc}z.ocninc.nc" ]]; then
+                    link_file "${system_prefix}.t${cyc}z.ocninc.nc" "${system_prefix}.t${cyc}z.mom6_increment.i006.nc"
+                fi
+            fi
+            # ice
+            if [[ -d "${cwd}/${dir}/${cyc}/${mem}/analysis/ice" ]]; then
+                cd "${cwd}/${dir}/${cyc}/${mem}/analysis/ice"
+                for ice_file in *.cice_model_anl.res.nc; do
+                    if [[ -f "${ice_file}" ]]; then
+                        prefix=$(echo "${ice_file}" | cut -d. -f1-2)
+                        link_file "${ice_file}" "${prefix}.analysis.cice_model.res.nc"
+                    fi
+                done
+            fi
+            # snow
+            if [[ -d "${cwd}/${dir}/${cyc}/${mem}/analysis/snow" ]]; then
+                cd "${cwd}/${dir}/${cyc}/${mem}/analysis/snow"
+                for snow_file in snowinc.*.sfc_data.tile1.nc; do
```

Comment:
We are naming the same files as snow_analsysis and snow_increment.  I think this should be 

for snow_file in ????????.??0000.sfc_data.tile1.nc  and explicitly exclude the snowinc files. 

@CoryMartin-NOAA @tsga can you help confirm?

Reply:
I believe it is fixed now.  I am going to run this through a copy of the ICSDIR.  I may need to recreate the links in the real ICSDIR as well.

Reply:
Fixed.